### PR TITLE
some entity work

### PR DESCRIPTION
**Pull request by eristhea**
I like to map

### CODE REVIEW

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/entity/projectile/BreezeWindCharge.mapping [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/unmapped/C_uqkupzdc net/minecraft/entity/projectile/BreezeWindCharge
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/entity/projectile/BreezeWindCharge.mapping]: #554 calls this `BreezeWindChargeEntity`, and it looks like other projectiles have the entity suffix

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/entity/data/TrackedDataHandlerRegistry.mapping [hunk 2]
```diff
@@ -35,3 +37,4 @@ CLASS net/minecraft/unmapped/C_fegtripb net/minecraft/entity/data/TrackedDataHan
 		ARG 0 handler
 	METHOD m_goylrcty get (I)Lnet/minecraft/unmapped/C_xrzqxzbg;
 		ARG 0 id
+	CLASS C_aaxqothd
```
Comment (IotaBread) [inline mappings/net/minecraft/entity/data/TrackedDataHandlerRegistry.mapping]: Empty mapping

**Review by IotaBread — APPROVED** [review]

File: mappings/net/minecraft/entity/data/TrackedDataHandlerRegistry.mapping (hunk 2 above)
Comment (eristhea) [reply mappings/net/minecraft/entity/data/TrackedDataHandlerRegistry.mapping]: lost the contents dropping invalid mappings, I'll re-map the class tonight

Comment (eristhea) [reply mappings/net/minecraft/entity/data/TrackedDataHandlerRegistry.mapping]: turns out the class just fully disappeared

**Pull request merged**